src/app/components/save-animation/save-animation.component.html

```diff
@@ -0,0 +1,6 @@
+<ng-template #content let-modal>
+  <div class="modal-body">
+    <div class="emoji"></div>
```

Comment:
:angry:

Reply:
just an emoji?

Reply:
@bucherfa Thought you were one of those internet guys where emoji equals comic sans ^^